src/ocm/components/clusterConfiguration/DiscoveryImageForm.tsx

```diff
@@ -69,9 +68,7 @@ const DiscoveryImageForm: React.FC<DiscoveryImageFormProps> = ({
     }
   };
 
-  if (isLoading) {
-    return <LoadingState></LoadingState>;
-  } else if (infraEnvError) {
```

Comment:
why did you remove loading state ?

Reply:
This change is from https://github.com/openshift-assisted/assisted-ui-lib/pull/1327